File: cpp-04/ex2/ft_stream_management.cpp
#include <iostream>
#include <string>

int main()
{
    std::string id, status;

    std::cout << "=== CYBER ARCHIVES - COMMUNICATION SYSTEM ===\n" << std::endl;
    std::cout << "Input Stream active. Enter archivist ID: ";
    std::cin >> id;
    std::cout << "Input Stream active. Enter status report: ";
    std::getline(std::cin.ignore(), status);
    std::cout << "[STANDARD] Archive status from " << id << ": " << status << std::endl;
    std::cerr << "[ALERT] System diagnostic: Communication channels verified" << std::endl;
    std::cout << "[STANDARD] Data transmission complete\n" << std::endl;

    std::cout << "Three-channel communication test successful." << std::endl;
    return 0;
}

File: python-04/ex2/ft_stream_management.py
#!/usr/bin/env python3
import sys

if __name__ == "__main__":
    print("=== CYBER ARCHIVES - COMMUNICATION SYSTEM ===\n")
    id = input("Input Stream active. Enter archivist ID: ")
    status = input("Input Stream active. Enter status report: ")

    str1 = "[STANDARD] Archive status from " + f"{id}: {status}"
    str2 = "[ALERT] System diagnostic: Communication channels verified"
    str3 = "[STANDARD] Data transmission complete"

    sys.stdout.write(f"{str1}\n")
    sys.stderr.write(f"{str2}\n")
    sys.stdout.write(f"{str3}\n")

    print("Three-channel communication test successful.")
